pyannote/core/annotation.py

```diff
@@ -472,6 +473,85 @@ def crop(self, support: Support, mode: CropMode = 'intersection'):
             else:
                 raise NotImplementedError("unsupported mode: '%s'" % mode)
 
+    def truncate(self, removed: Support, mode: CropMode = 'intersection') \
+            -> 'Annotation':
+        """Removes all segments or parts of segments that
+        overlap the `removed` argument.
+
+        Parameters
+        ----------
+        support : Segment or Timeline
+            If `support` is a `Timeline`, its support is used.
+        mode : {'strict', 'loose', 'intersection'}, optional
+            Controls how segments that are not fully included in `removed` are
+            handled. 'strict' mode only removes fully included segments. 'loose'
+            mode removes any intersecting segment. 'intersection' mode removes any
+            intersecting segment but replace them by their actual intersection.
```

Comment:
It does not replace them by their intersection. 
It replace*s* them by the part that does not intersect, right?

Reply:
Would be nice to add a visual representation of the effect of each mode (like what you did for `overlaps`).